### ISSUE TITLE
manual/javaGuide/main/async/JavaComet.md

### ISSUE THREAD

**Issue by garbagetown** [playframework-ja/translation-project#938]
- https://github.com/playframework-ja/translation-project/tree/2.3.5/manual/javaGuide/main/async/JavaComet.md
- diffs between [2.2.0/manual/javaGuide/main/async/JavaComet.md](https://github.com/playframework-ja/translation-project/tree/2.2.0/manual/javaGuide/main/async/JavaComet.md) and [2.3.5/manual/javaGuide/main/async/JavaComet.md](https://github.com/playframework-ja/translation-project/tree/2.3.5/manual/javaGuide/main/async/JavaComet.md).

```
--- /Users/garbagetown/Desktop/2.2.0/manual/javaGuide/main/async/JavaComet.md   2013-09-19 10:11:22.000000000 +0900
+++ //Users/garbagetown/Desktop/2.3.5/manual/javaGuide/main/async/JavaComet.md  2014-07-28 12:26:38.000000000 +0900
@@ -1,3 +1,4 @@
+<!--- Copyright (C) 2009-2013 Typesafe Inc. <http://www.typesafe.com> -->
 # Comet sockets

 ## Using chunked responses to create Comet sockets
@@ -6,7 +7,11 @@

 Let’s write a first proof-of-concept: create an enumerator generating `<script>` tags calling the browser `console.log` function:

-@[manual](code/javaguide/async/JavaComet.java)
+Java
+: @[manual](code/javaguide/async/JavaComet.java)
+
+Java 8
+: @[manual](java8code/java8guide/async/JavaComet.java)

 If you run this action from a web browser, you will see the three events logged in the browser console.

@@ -18,13 +23,21 @@

 Let’s just rewrite the previous example to use it:

-@[comet](code/javaguide/async/JavaComet.java)
+Java
+: @[comet](code/javaguide/async/JavaComet.java)
+
+Java 8
+: @[comet](java8code/java8guide/async/JavaComet.java)

 ## The forever iframe technique

 The standard technique to write a Comet socket is to load an infinite chunked comet response in an iframe and to specify a callback calling the parent frame:

-@[forever-iframe](code/javaguide/async/JavaComet.java)
+Java
+: @[forever-iframe](code/javaguide/async/JavaComet.java)
+
+Java 8
+: @[forever-iframe](java8code/java8guide/async/JavaComet.java)

 With an HTML page like:


```

**Comment by garbagetown** [comment]
:hand:

**Comment by garbagetown** [comment]
merged #1127
